boom! testing metaheuristics-0.0.12 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! c4cf1dc28f0d2921498ae501e381f7257fd8c4a8a6d56279e1e46085d916587f
boom! running `docker start -a c4cf1dc28f0d2921498ae501e381f7257fd8c4a8a6d56279e1e46085d916587f`
kablam!    Compiling metaheuristics v0.0.12 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.47 secs
boom! running `docker rm -f c4cf1dc28f0d2921498ae501e381f7257fd8c4a8a6d56279e1e46085d916587f`
blam! c4cf1dc28f0d2921498ae501e381f7257fd8c4a8a6d56279e1e46085d916587f
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! bd25421c9a31bc6601bfec2340f33851372e42466331031841f4ee7aafc82716
boom! running `docker start -a bd25421c9a31bc6601bfec2340f33851372e42466331031841f4ee7aafc82716`
kablam!    Compiling metaheuristics v0.0.12 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 1.42 secs
boom! running `docker rm -f bd25421c9a31bc6601bfec2340f33851372e42466331031841f4ee7aafc82716`
blam! bd25421c9a31bc6601bfec2340f33851372e42466331031841f4ee7aafc82716
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! 45c493555366942c1adc99f42b152a0d0effaac54268006eab905695e7bbda1c
boom! running `docker start -a 45c493555366942c1adc99f42b152a0d0effaac54268006eab905695e7bbda1c`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/metaheuristics-1ec4fbc3b8425e1a
blam! 
kablam!    Doc-tests metaheuristics
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
blam! 
blam! running 14 tests
blam! test Metaheuristics::generate_candidate_0 ... FAILED
blam! test Metaheuristics::rank_candidate_0 ... FAILED
blam! test Metaheuristics::clone_candidate_0 ... FAILED
blam! test Metaheuristics::tweak_candidate_0 ... FAILED
blam! test hill_climbing::random_restarts::solve_0 ... FAILED
blam! test _0 ... FAILED
blam! test hill_climbing::random_restarts_0 ... FAILED
blam! test hill_climbing::solve_0 ... FAILED
blam! test random_search::solve_0 ... FAILED
blam! test hill_climbing_0 ... FAILED
blam! test simulated_annealing::solve_0 ... FAILED
blam! test random_search_0 ... FAILED
blam! test simulated_annealing_0 ... FAILED
blam! test simulated_annealing_1 ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- Metaheuristics::generate_candidate_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:2:18
blam!   |
blam! 2 |  let candidate = problem.generate_candidate();
blam!   |                  ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- Metaheuristics::rank_candidate_0 stdout ----
blam! 	error: inclusive range with no end
blam!  --> <anon>:4:2
blam!   |
blam! 4 |  }
blam!   |  ^
blam!   |
blam!   = help: inclusive ranges must be bounded at the end (`...b` or `a...b`)
blam! 
blam! error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:2:5
blam!   |
blam! 2 |  if problem.rank_candidate(&new_candidate) > problem.rank_candidate(&old_candidate) {
blam!   |     ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `new_candidate` in this scope
blam!  --> <anon>:2:29
blam!   |
blam! 2 |  if problem.rank_candidate(&new_candidate) > problem.rank_candidate(&old_candidate) {
blam!   |                             ^^^^^^^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:2:46
blam!   |
blam! 2 |  if problem.rank_candidate(&new_candidate) > problem.rank_candidate(&old_candidate) {
blam!   |                                              ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `old_candidate` in this scope
blam!  --> <anon>:2:70
blam!   |
blam! 2 |  if problem.rank_candidate(&new_candidate) > problem.rank_candidate(&old_candidate) {
blam!   |                                                                      ^^^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- Metaheuristics::clone_candidate_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 |  let new_candidate = problem.clone_candidate(&old_candidate);
blam!   |                      ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `old_candidate` in this scope
blam!  --> <anon>:2:47
blam!   |
blam! 2 |  let new_candidate = problem.clone_candidate(&old_candidate);
blam!   |                                               ^^^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- Metaheuristics::tweak_candidate_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 |  let new_candidate = problem.tweak_candidate(&old_candidate);
blam!   |                      ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `old_candidate` in this scope
blam!  --> <anon>:2:47
blam!   |
blam! 2 |  let new_candidate = problem.tweak_candidate(&old_candidate);
blam!   |                                               ^^^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- hill_climbing::random_restarts::solve_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:4:10
blam!   |
blam! 4 |     &mut problem,
blam!   |          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:5:5
blam!   |
blam! 5 |     runtime,
blam!   |     ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `probability` in this scope
blam!  --> <anon>:6:5
blam!   |
blam! 6 |     probability
blam!   |     ^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- _0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:59
blam!   |
blam! 3 |  let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                           ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:68
blam!   |
blam! 3 |  let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                                    ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- hill_climbing::random_restarts_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:4:10
blam!   |
blam! 4 |     &mut problem,
blam!   |          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:5:5
blam!   |
blam! 5 |     runtime,
blam!   |     ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `probability` in this scope
blam!  --> <anon>:6:5
blam!   |
blam! 6 |     probability
kablam! error: test failed
blam!   |     ^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- hill_climbing::solve_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:58
blam!   |
blam! 3 | let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:67
blam!   |
blam! 3 | let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                                   ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- random_search::solve_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:58
blam!   |
blam! 3 | let solution = metaheuristics::random_search::solve(&mut problem, runtime);
blam!   |                                                          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:67
blam!   |
blam! 3 | let solution = metaheuristics::random_search::solve(&mut problem, runtime);
blam!   |                                                                   ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- hill_climbing_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:58
blam!   |
blam! 3 | let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:67
blam!   |
blam! 3 | let solution = metaheuristics::hill_climbing::solve(&mut problem, runtime);
blam!   |                                                                   ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- simulated_annealing::solve_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:64
blam!   |
blam! 3 | let solution = metaheuristics::simulated_annealing::solve(&mut problem, runtime);
blam!   |                                                                ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:73
blam!   |
blam! 3 | let solution = metaheuristics::simulated_annealing::solve(&mut problem, runtime);
blam!   |                                                                         ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- random_search_0 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:58
blam!   |
blam! 3 | let solution = metaheuristics::random_search::solve(&mut problem, runtime);
blam!   |                                                          ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:67
blam!   |
blam! 3 | let solution = metaheuristics::random_search::solve(&mut problem, runtime);
blam!   |                                                                   ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- simulated_annealing_0 stdout ----
blam! 	error[E0425]: cannot find function `P` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | P(t) = e^(-10*(t^3))
blam!   | ^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `t` in this scope
blam!  --> <anon>:2:3
blam!   |
blam! 2 | P(t) = e^(-10*(t^3))
blam!   |   ^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `e` in this scope
blam!  --> <anon>:2:8
blam!   |
blam! 2 | P(t) = e^(-10*(t^3))
blam!   |        ^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `t` in this scope
blam!  --> <anon>:2:16
blam!   |
blam! 2 | P(t) = e^(-10*(t^3))
blam!   |                ^ not found in this scope
blam! 
blam! error[E0070]: invalid left-hand side expression
blam!  --> <anon>:2:1
blam!   |
blam! 2 | P(t) = e^(-10*(t^3))
blam!   | ^^^^^^^^^^^^^^^^^^^^ left-hand of expression not valid
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! ---- simulated_annealing_1 stdout ----
blam! 	error[E0425]: cannot find value `problem` in this scope
blam!  --> <anon>:3:64
blam!   |
blam! 3 | let solution = metaheuristics::simulated_annealing::solve(&mut problem, runtime);
blam!   |                                                                ^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `runtime` in this scope
blam!  --> <anon>:3:73
blam!   |
blam! 3 | let solution = metaheuristics::simulated_annealing::solve(&mut problem, runtime);
blam!   |                                                                         ^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! 
blam! 
blam! failures:
blam!     Metaheuristics::clone_candidate_0
blam!     Metaheuristics::generate_candidate_0
blam!     Metaheuristics::rank_candidate_0
blam!     Metaheuristics::tweak_candidate_0
blam!     _0
blam!     hill_climbing::random_restarts::solve_0
blam!     hill_climbing::random_restarts_0
blam!     hill_climbing::solve_0
blam!     hill_climbing_0
blam!     random_search::solve_0
blam!     random_search_0
blam!     simulated_annealing::solve_0
blam!     simulated_annealing_0
blam!     simulated_annealing_1
blam! 
blam! test result: FAILED. 0 passed; 14 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f 45c493555366942c1adc99f42b152a0d0effaac54268006eab905695e7bbda1c`
blam! 45c493555366942c1adc99f42b152a0d0effaac54268006eab905695e7bbda1c
